### ISSUE TITLE
Site map reorganisation 

### ISSUE THREAD

**Issue by moneroguides** [monero-project/monero-site#1900]
It appears that many have complaints about the current site, including myself. Here is my attempt and a restructure to help drive change.

Feedback from monero-ux:monero.social and monero-site:haveno.network went into what you see here.

![sitemapv4-4](https://user-images.githubusercontent.com/93890007/145279836-a67651ea-8068-437a-af4a-a70b83a61353.png)

**Comment by erciccione** [comment]
Note that this would be an important change that will definitely require a focused discussions. Also rises the question of what content should go on Getmonero.

**Comment by moneroguides** [comment]
Indeed. In that case i suggest a minuted meeting be arranged to take this further.

**Comment by erciccione** [comment]
As i was saying yesterday. This issue needs to be expanded with more details.

**Comment by moneroguides** [comment]
How much detail would you like to see with regard to this issue? For example, would you like to see a mock up of the 'new pages' and website?

**Comment by erciccione** [comment]
> How much detail would you like to see with regard to this issue?

An explanation of what changes you are suggesting and why would be a good start. Separating between changes to the menu and changes to the content of the website would also help speed things up.

**Comment by moneroguides** [comment]
I will carry on with this then :) 
![image](https://user-images.githubusercontent.com/93890007/146289999-199e5af7-9a31-4f3c-97e1-8b2716b6def7.png)

**Comment by moneroguides** [comment]
https://github.com/moneroguides/getmonero.org

I have created a new repo for ideas and plan. Will continue with that for now.
